### PR TITLE
Manage eks workers' root_block_device

### PR DESCRIPTION
**Pull request by ozbillwang**
# PR o'clock

## Description

for #57 to manage eks workers' root volume size, type and iops (if volume type is iops)

### Checklist

- [x] `terraform fmt` and `terraform validate` both work from the root and `examples/eks_test_fixture` directories (look in CI for an example)
- [x] Tests for the changes have been added and passing (for bug fixes/features)
- [x] Test results are pasted in this PR (in lieu of CI)
- [x] Docs have been updated using `terraform-docs` per `README.md` instructions
- [x] I've added my change to CHANGELOG.md
- [x] Any breaking changes are highlighted above

### CODE REVIEW

**Comment by brandoconnor** [comment]
Let me know when this is ready to ship. I've added WIP to the title but that can be removed when this work is ready for review.

**Comment by ozbillwang** [comment]
@brandoconnor

It can be reviewed already

File: variables.tf [hunk 1]
```diff
@@ -85,6 +85,9 @@ variable "workers_group_defaults" {
     asg_max_size         = "3"           # Maximum worker capacity in the autoscaling group.
     asg_min_size         = "1"           # Minimum worker capacity in the autoscaling group.
     instance_type        = "m4.large"    # Size of the workers instances.
+    root_volume_size     = "20"          # root volume size of workers instances.
```
Comment (brandoconnor) [inline variables.tf]: Should have asked before I merged: Are these values the defaults that worker ASGs received before this change?

Comment (ozbillwang) [reply variables.tf]: Not sure. When I start EKS worker as `m5.xlarge`, its root disk size is 20GB, has no chance to test with other instance. 

But if AWS designs the EKS workers and set 20GB with gp2, it should be very reasonable and I also think it works for most case. 

But if you need double check, I can try to start other instance type or if anyone can directly give the data if they already run with other instance types.

**Comment by ozbillwang** [comment]
By the way, do you need me add details in README (in main.tf) to explain how to use the map variable `worker_groups`, it becomes an important feature for this module now.

File: variables.tf (hunk 1 above)
Comment (ozbillwang) [reply variables.tf]: @brandoconnor 

I can confirm the default root size now: 

https://github.com/awslabs/amazon-eks-ami/blob/master/eks-worker-al2.json#L22

above repo is managed by AWS EKS team and use packer to make worker AMI images.

**Comment by brandoconnor** [comment]
Nice!

**Pull request merged**